tests/Oryx.BuildImage.Tests/Node/NodeJSSampleAppsTestBase.cs

```diff
@@ -24,5 +28,43 @@ public class NodeJSSampleAppsTestBase : SampleAppsTestBase
             }))
         {
         }
+
+        [Fact]
```

Comment:
@Wali I noticed this test doesn't run in the pipeline since there's no assigned category. Could you add a category to this function and move it outside the TestBase with other tests. Example: https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsOtherTests.cs#L124

Reply:
Please also merge with `main`  before making the change since BuildImage tests have noticeable changes